contracts/ens-offchain-resolver/IResolverService.sol

```diff
@@ -3,6 +3,9 @@
 pragma solidity 0.8.13;
 
 interface IResolverService {
+    /**
+     * @notice This function selector is encoded with name and data to pass to the offchain gateway.
```

Comment:
Same here please @petejkim - let me know if there is a better comment to add here.

Reply:
```
    /**
     * @notice Function interface for the lookup function supported by the off-chain gateway.
     * @dev This function is executed off-chain by the off-chain gateway.
     * @param name DNS-encoded name to resolve.
     * @param data ABI-encoded data for the underlying resolution function (e.g. addr(bytes32), text(bytes32,string)).
     * @return result ABI-encode result of the lookup.
     * @return expires Time at which the signature expires.
     * @return sig A signer's signature authenticating the result.
     */
```